src/jsoncharutils.h

```diff
@@ -678,641 +678,641 @@ static const components power_of_ten_components[] = {
-                                 0x7e2fa67c7a658892,
-                                 0xddbb901b98feeab7,
-                                 0x552a74227f3ea565,
-                                 0xd53a88958f87275f,
-                                 0x8a892abaf368f137,
-                                 0x2d2b7569b0432d85,
-                                 0x9c3b29620e29fc73,
-                                 0x8349f3ba91b47b8f,
-                                 0x241c70a936219a73,
-                                 0xed238cd383aa0110,
-                                 0xf4363804324a40aa,
-                                 0xb143c6053edcd0d5,
-                                 0xdd94b7868e94050a,
-                                 0xca7cf2b4191c8326,
-                                 0xfd1c2f611f63a3f0,
-                                 0xbc633b39673c8cec,
-                                 0xd5be0503e085d813,
-                                 0x4b2d8644d8a74e18,
-                                 0xddf8e7d60ed1219e,
-                                 0xcabb90e5c942b503,
-                                 0x3d6a751f3b936243,
-                                 0xcc512670a783ad4,
-                                 0x27fb2b80668b24c5,
-                                 0xb1f9f660802dedf6,
-                                 0x5e7873f8a0396973,
-                                 0xdb0b487b6423e1e8,
-                                 0x91ce1a9a3d2cda62,
-                                 0x7641a140cc7810fb,
-                                 0xa9e904c87fcb0a9d,
-                                 0x546345fa9fbdcd44,
-                                 0xa97c177947ad4095,
-                                 0x49ed8eabcccc485d,
-                                 0x5c68f256bfff5a74,
-                                 0x73832eec6fff3111,
-                                 0xc831fd53c5ff7eab,
-                                 0xba3e7ca8b77f5e55,
-                                 0x28ce1bd2e55f35eb,
-                                 0x7980d163cf5b81b3,
-                                 0xd7e105bcc332621f,
-                                 0x8dd9472bf3fefaa7,
-                                 0xb14f98f6f0feb951,
-                                 0x6ed1bf9a569f33d3,
-                                 0xa862f80ec4700c8,
-                                 0xcd27bb612758c0fa,
-                                 0x8038d51cb897789c,
-                                 0xe0470a63e6bd56c3,
-                                 0x1858ccfce06cac74,
-                                 0xf37801e0c43ebc8,
-                                 0xd30560258f54e6ba,
-                                 0x47c6b82ef32a2069,
-                                 0x4cdc331d57fa5441,
-                                 0xe0133fe4adf8e952,
-                                 0x58180fddd97723a6,
-                                 0x570f09eaa7ea7648};
-
+const uint64_t mantissa_128[] = {
```

Comment:
@lemire I actually like this change for readability. Large indents are jarring. It's entirely possible I made this change when I was being more conservative about style at the beginning of collaboration :)

Reply:
@jkeiser This is the LLVM style as interpreted by clang-format...

https://github.com/simdjson/simdjson/blob/master/.clang-format

We can change that, of course.

Reply:
Oh, if there is a style formatter already I won't impose; I'd even rather enforce it :)

Reply:
We have not been using it much but it is there... 

```
$ ls style/
clang-format-check.sh clang-format.sh       run-clang-format.py
```

Reply:
To achieve the reformatting needed, we need to add a trailing comma to the long array prior to formatting...

<img width="1662" alt="Screen Shot 2020-04-24 at 5 35 40 PM" src="https://user-images.githubusercontent.com/391987/80258814-1f5f4900-8652-11ea-9a78-ea6084bd9e48.png">